app/pages/AppContainer.js

```diff
@@ -60,23 +60,23 @@ export class UnconnectedAppContainer extends Component {
 
   render() {
     return (
-      <BodyClassName className={getCustomizationClassName()}>
-        <DocumentTitle title="Varaamo">
-          <div className="app">
-            <Header location={this.props.location}>
-              <Favicon />
-              <TestSiteMessage />
-            </Header>
-            <div className="app-content">
-              <Grid>
-                <Notifications />
-              </Grid>
-              {this.props.children}
-            </div>
-            <Footer />
-          </div>
-        </DocumentTitle>
-      </BodyClassName>
+      <div className={classNames('app', getCustomizationClassName())}>
```

Comment:
Are you 100% sure this is enough, because if there are any styles that depend on body class name then this most likely doesn't work?

Reply:
We just have to be 100% sure that Vantaa/Espoo site styles work like they use to work.

Reply:
@rBoost: I tested it under Espoo/Vantaa `varaamo.vantaa.fi`, remove the class from body and move it to app, everything still work after that

Reply:
ok, good. 👍